crates/kitsune_p2p/kitsune_p2p/src/spawn/actor/space.rs

```diff
@@ -1555,12 +1555,19 @@ impl Space {
             match network_type {
                 NetworkType::QuicMdns => tracing::warn!("NOT publishing leaves to mdns"),
                 NetworkType::QuicBootstrap => {
-                    kitsune_p2p_bootstrap_client::put(
+                    match kitsune_p2p_bootstrap_client::put(
```

Comment:
I don't think returning an error here is useful. This is the agent leave and if we fail to publish that to the bootstrap server then it doesn't affect us that much. It would be useful to other nodes but they'll figure out that we're gone soon enough